octoprint_mrbeam/migration/Mig006.py

```diff
@@ -39,106 +36,24 @@ def shouldrun(cls, beamos_version):
 
         overrides the current behaviour as this migration should run if the log file contains the "octoprint.plugins.mrbeam.analytics.usage - ERROR - No job time found in {}, returning 0" error
         """
```

Comment:
Please remove the above:

COMMAND_TO_GET_LOGS = 'grep -r "octoprint.plugins.mrbeam.analytics.usage - ERROR - No job time found in {}" /home/pi/.octoprint/logs/'
    COMMAND_TO_CHECK_IF_VERSION_WAS_PRESENT = 'grep -a -e "Mr Beam Laser Cutter (0.15.0.post0) = /home/pi/oprint/local/lib/python2.7/site-packages/octoprint_mrbeam" -e "Mr Beam Laser Cutter (0.15.0) = /home/pi/oprint/local/lib/python2.7/site-packages/octoprint_mrbeam" /home/pi/.octoprint/logs/*'
    USAGE_DATA_FILE_PATH = "/home/pi/.octoprint/analytics/usage.yaml"

Reply:
and modify the comments under shouldrun method

Reply:
ok